### PR TITLE
Adds A Man of Weh poster

### PR DESCRIPTION
**Pull request by spanky-spanky**
## About the PR
Adds A Man of Weh poster by eli_sarek to the random poster pool.

## Why / Balance
Quality Weh art!

## Media

image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: Add Man of Weh poster, by eli_sarek.

### CODE REVIEW

File: Resources/Textures/Harmony/Structures/Wallmounts/posters.rsi/meta.json [hunk 1]
```diff
@@ -13,5 +13,8 @@
         {
             "name": "harmony_poster2_legit"
         }
```
Comment (FluffMe) [inline Resources/Textures/Harmony/Structures/Wallmounts/posters.rsi/meta.json:15]: Missing comma

**Review by FluffMe — CHANGES_REQUESTED** [review]
One missing comma on line 15 of meta.json

Fails linter and rsi checks due to invalid JSON.

File: Resources/Textures/Harmony/Structures/Wallmounts/posters.rsi/meta.json (hunk 1 above)
Comment (spanky-spanky) [reply Resources/Textures/Harmony/Structures/Wallmounts/posters.rsi/meta.json:15]: What the hell is a comma

will fix

Comment (FluffMe) [reply Resources/Textures/Harmony/Structures/Wallmounts/posters.rsi/meta.json:15]: comma is `,` !!!

**Review by FluffMe — APPROVED** [review]
LGTM!

**Pull request merged**